libs/common/src/test-helpers/mock-data-backend/portal-facility-amendment.ts

```diff
@@ -26,6 +26,7 @@ export const aPortalFacilityAmendment = ({ status = PORTAL_AMENDMENT_STATUS.DRAF
   createdAt: getUnixTime(new Date()),
   updatedAt: getUnixTime(new Date()),
   status,
+  referenceNumber: `${new ObjectId().toString()}-01`,
```

Comment:
Should this not be Facility ID from number generator i.e. `ukefFacilityId-01` instead of Mongo Facility ID?